CMakePresets.json

```diff
@@ -83,6 +83,16 @@
         "USE_JEMALLOC": "Off"
       }
     },
+    {
+      "hidden": true,
+      "name": "gcov",
+      "cacheVariables": {
+        "USE_COVERAGE": "On",
+        "CMAKE_CXX_FLAGS": "-fno-stack-protector -fprofile-arcs -ftest-coverage",
+        "CMAKE_C_FLAGS": "-fno-stack-protector -fprofile-arcs -ftest-coverage",
```

Comment:
```suggestion
```
I now think it is best to leave the option in `CMakeLists.txt`, but certainly to remove these two lines.

Rationale: This way we could maybe switch coverage on only for *our* code, as I believe at the moment we just enable coverage for *everything*.